jreleaser.yml

```diff
@@ -0,0 +1,76 @@
+# Hooks that will run on the CI to generate the summary of the steps
+hooks:
+  condition: '"{{ Env.CI }}" == true'
+  script:
+    before:
+      - filter:
+          includes: ['session']
+        run: |
+          echo "### {{command}}" >> $GITHUB_STEP_SUMMARY
+          echo "| Step | Outcome |" >> $GITHUB_STEP_SUMMARY
+          echo "| ---- | ------- |" >> $GITHUB_STEP_SUMMARY
+    success:
+      - filter:
+          excludes: ['session']
+        run: 'echo "| {{event.name}} | :white_check_mark: |" >> $GITHUB_STEP_SUMMARY'
+      - filter:
+          includes: ['session']
+        run: echo "" >> $GITHUB_STEP_SUMMARY
+    failure:
+      - filter:
+          excludes: ['session']
+        run: 'echo "| {{event.name}} | :x: |" >> $GITHUB_STEP_SUMMARY'
+      - filter:
+          includes: ['session']
+        run: |
+          echo "" >> $GITHUB_STEP_SUMMARY
+          echo "### Failure" >> $GITHUB_STEP_SUMMARY
+          echo "\`\`\`" >> $GITHUB_STEP_SUMMARY
+          echo "{{event.stacktrace}}\`\`\`" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_STEP_SUMMARY
+
+# Project configuration
+project:
+  name: airlift
+  description: Airlift is a framework for building REST services in Java.
+  license: Apache-2
+  java:
+    groupId: io.airlift
+    multiProject: true
+
+# Ordered as defined in the https://jreleaser.org/guide/latest/concepts/workflow.html#_full_release
+signing:
+  active: ALWAYS
+  armored: true
+
+# Deploy to OSSRH
+deploy:
+  maven:
+    pomchecker:
+      failOnWarning: false # We don't want to fail the build on warnings
+      failOnError: false # We don't want to fail the build on errors
+    nexus2:
+      maven-central:
+        active: ALWAYS
+        url: https://oss.sonatype.org/service/local
+        snapshotUrl: https://oss.sonatype.org/content/repositories/snapshots/
+        closeRepository: true
+        javadocJar: false # Not every module has javadoc (packaging)
+        releaseRepository: false # Switch to true to release to Central
```

Comment:
This is now configure not to release a staging repository. This is done by hand in the OSS UI